content/en/docs/Contribute/_index.md

```diff
@@ -1,35 +1,22 @@
 <!--
 ---
-linkTitle: "Contributing to Tekton Documentation"
-weight: 1
+title: "Contribute to Documentation"
+linkTitle: "Contribute to Documentation"
+weight: 11
+description: > 
+  Contribution guidelines
 ---
 -->
-# Contributing to Tekton Documentation
 
 If you would like to contribute to Tekton documentation, we’re happy to have your help!
 Anyone can contribute, whether you’re new to the project or you’ve been around a long time,
 and whether you self-identify as a developer, an end user, or someone who just can’t stand
 seeing typos.
 
-This guide covers the following topics:
-
-- [Contribution types](#contribution-types)
-- [Contribution process](#contribution-process)
-- [Requesting a documentation improvement](#requesting-a-documentation-improvement)
-- [Submitting a documentation change](#submitting-a-documentation-change)
-- [I need help!](#i-need-help)
-
-Before you begin contributing, you should also read the following:
-
-- [Content guidelines for Tekton documentation](doc-con-content.md)
-- [Formatting conventions for Tekton documentation](doc-con-formatting.md)
-- [Writing high quality documentation for Tekton](doc-con-writing.md)
-
 ## Contribution types
 
 You can request an improvement by filing an issue or update the documentation yourself by
 filing a pull request against the relevant [Tekton repository](https://github.com/tektoncd).
-Assign your issue or pull request to @sergetron, Tekton's technical writer, for triage.

```

Comment:
There's another `@sergetron` a few paragraphs below in the 'Contribution process' section:

`Assign the pull request and if applicable, the accompanying issue to @sergetron, Tekton's
technical writer.`